docs/integration_resources/resources/usd_tutorials/usd_tutorial_03.md

```diff
@@ -0,0 +1,188 @@
+    <figcaption><b>Figure 18</b>: Added the 'WorldGet' node to the board connected to the 'RefSphere2StageGet' and 'RefSphere2StageSave' nodes.</figcaption>
+</figure>
+
+19. Add a new **<tt>UsdPrimDisplayColorSet</tt>** node to the board and rename it to `WorldDisplayColorSet`. Connect the **prim** output plug from the `WorldGet` node to the **prim** input plug of the new node. Set `[1, 0, 0]` as the value for the **color** input plug, `[]` as the value for the **indices** input plug and `constant` as the value for the **interpolation** input plug of the new node. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_03/step_19_t03.png" alt="Added the 'WorldDisplayColorSet' node to the board connected to the 'WorldGet' node.">
+    <figcaption><b>Figure 19</b>: Added the 'WorldDisplayColorSet' node to the board connected to the 'WorldGet' node.</figcaption>
+</figure>
+
+20. Add another **<tt>UsdPrimStageGet</tt>** node to the board and rename it to `WorldStageGet`. Connect the **prim** output plug from the `WorldDisplayColorSet` node to the **prim** input plug of the new node. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_03/step_20_t03.png" alt="Added the 'WorldStageGet' node to the board connected to the 'WorldDisplayColorSet' node.">
+    <figcaption><b>Figure 20</b>: Added the 'WorldStageGet' node to the board connected to the 'WorldDisplayColorSet' node.</figcaption>
+</figure>
+
+21. Add another **<tt>UsdStageSave</tt>** node to the board and rename it to `WorldStageSave`. Connect the **stage** output plug from the `WorldStageGet` node to the **stage** input plug of the new node. Leave empty the value for the **filepath** input plug, doing this, the original file will be overriden. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_03/step_21_t03.png" alt="Added the 'WorldStageSave' node to the board connected to the 'WorldStageGet' node.">
+    <figcaption><b>Figure 21</b>: Added the 'WorldStageSave' node to the board connected to the 'WorldStageGet' node.</figcaption>
+</figure>
+
+22. Finally, for *flattening* the resulting stage. Following the Pixar's documentation page, *flattening* means producing a single layer of scene description that contains the final “composed data” from a set of composed layers, and retains no composition operators such as references, payloads, inherits, variants, sublayers, and activations. For doing this, add a new **<tt>UsdStageFlatten</tt>** node to the board and rename it to `StageFlatten`. Connect the **stage** output plug from the `WorldStageGet` node to the **stage** input plug of the new node. Leave the **layer** input plug with the default value `None` set. Also, connect the **out-trigger** output plug from the `WorldStageSave` node to the **in-trigger** input plug of the new node. The board should look like the following:
```

Comment:
No need to explain it. If you want you can add a hyperlink to the meaning in Pixar's documentation.

Try to avoid starting sentences with Also . You can use additionally or finally  instead.
```suggestion
22. To flatten the resulting stage, add a new **<tt>UsdStageFlatten</tt>** node to the board and rename it to `StageFlatten`. Connect the **stage** output plug from the `WorldStageGet` node to the **stage** input plug of the new node. Leave the **layer** input plug with the default value `None` set. Finally, connect the **out-trigger** output plug from the `WorldStageSave` node to the **in-trigger** input plug of the new node. The board should look like the following:
```